### PR TITLE
#122/style message form

### PR DESCRIPTION
**Pull request by nichgalzin**
# Checklist:

- [x] My code follows the style guidelines of this project
- [x] I have performed a self-review on my code
- [x] I have made corresponding changes to the documentation (if any were needed)
- [x] My changes generate no new warnings or errors
- [ ] I have added tests that prove my fix is effective or that my feature works
- [x] New and previously existing unit tests pass locally with my changes
- [x] Any dependent changes have been merged and published in downstream modules

# Description

**Relates #[122] / Closes #[122]**
- Styling improvements to the message form component

### Files changed

- app/globals.css -- global styling change on focused elements to use orange highlight
- app/styles/messaging-styles.css -- adjusted height of the conversations container 
- components/icons/PaperPlaneIcon.tsx -- refactored component to allow props for size
- components/messaging/ConversationWrapper.tsx -- styling
- components/messaging/MessageForm.tsx -- styling and dynamic size for text area

### UI changes

If this PR includes changes to the UI include a **screengrab** of the alterations in review.
Screenshot 2024-03-19 at 16 10 21

Screenshot 2024-03-19 at 16 11 24

# Tests

Include a description of any tests added - if none have been added explain why.

No test added as this PR is styling related.

### CODE REVIEW

**Comment by eliazzo** [comment]
This looks really nice!

One thing, I'm not sure about the double border when typing a message. It think might look better to change the original border colour instead of introducing another.

Screenshot 2024-03-20 at 11 34 52

I also think the send button should be smaller as per the figma
Screenshot 2024-03-20 at 11 39 34

File: app/styles/messaging-styles.css [hunk 1]
```diff
@@ -4,14 +4,14 @@
 }
 
 .conversation-height {
-  height: calc(100vh - 225px);
+  height: calc(100vh - 175px);
```
Comment (camelPhonso) [inline app/styles/messaging-styles.css:7]: I wonder if we could refer to the height of the form directly here for maintenance sake 🤔 

I've not done this in CSS before, but presumably we can just have the `calc()` fetch a variable - maybe we can set it so that if the height of the form changes in future, developers don't need to find and manually change this value too, but the calculation considers the new value of it's height

**Review by camelPhonso — COMMENTED** [review]
Looks pretty cool ✨ 

Made a suggestion that I am not sure how easy will be to implement but is worth thinking about for a second

File: app/globals.css [hunk 2]
```diff
@@ -19,9 +19,12 @@ body {
   @apply bg-background font-inter text-primaryGray;
 }
 
-*:focus {
-  outline: 2px solid #FF9E5E; /* Custom color */
-  outline-offset: 2px;
+/* * {
+  border: 1px solid red;
+} */
```
Comment (camelPhonso) [inline app/globals.css:24]: 👻

**Review by camelPhonso — CHANGES_REQUESTED** [review]
Looks great - just needs some ghost busters

**Review by camelPhonso — APPROVED** [review]
Awesome possum. Happy to merge this
